File: benchmark/results/ts-multilang/run-003/b-baseline/output/src/domain/billing/services/TaxCalculator.ts
import { Invoice } from "../aggregates/Invoice";

/** TaxCalculator domain service for the billing domain. */
export class TaxCalculator {
  create(input: Partial<Invoice>): Invoice {
    void input;
    throw new Error("not implemented");
  }

  findById(id: string): Invoice | null {
    void id;
    throw new Error("not implemented");
  }

  list(limit: number): Invoice[] {
    void limit;
    throw new Error("not implemented");
  }

  delete(id: string): void {
    void id;
    throw new Error("not implemented");
  }
}

File: benchmark/results/python-invocation-script/run-005/a-mcp/output/billing/services/tax_calculator.py
from billing.aggregates.invoice import Invoice

"""TaxCalculator service."""
class TaxCalculator:
    def create(self, input: Invoice) -> Invoice:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Invoice | None:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> list[Invoice]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

